src/pyhf/contrib/viz/brazil.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+
+
+def plot_results(ax, mutests, tests, test_size=0.05):
+    cls_obs = np.array([test[0] for test in tests]).flatten()
+    cls_exp = [np.array([test[1][i] for test in tests]).flatten() for i in range(5)]
+    ax.plot(mutests, cls_obs, c='k')
```

Comment:
I'd suggest using verbose color names
```suggestion
    ax.plot(mutests, cls_obs, c='black')
```